INSTALL.md

```diff
@@ -479,6 +482,34 @@ default.debug=true
 EOL
 ```
 
+## 3.12. Building on Arch Linux
+
+### 3.12.1. Install Build Dependencies
+
+```bash
+sudo pacman -S pyqt-builder gsl gpsbabel postgresql sip libzip qtkeychain-qt5 qca-qt5 qscintilla-qt5 expat qwt gdal \
+spatialindex sqlite geos proj qt5 libspatialite cmake gcc flex bison
+```
+
+### 3.12.2. Compiling
+
+In the root of the repository, at any tag or branch you choose:
+
+Keep in mind, this will destroy any contents of your build directory if you have one.
+
+```bash
+if [ -d ./build/ ]; then rm -rfv ./build/*; else mkdir -v build; fi
+cd build
+cmake ..
+make
```

Comment:
`cmake --build .` will use all cores. :>

Reply:
Is this something that should be documented per distro?
One single reference would be easier to keep up to date and could also go more in depth (e.g. explaining `CMAKE_BUILD_TPYE`)

My favorite

```
cmake -GNinja -S . -B build
cmake --build build
```